### ISSUE TITLE
Music pauses on Android when loading a WebRTC card with muted sound

### ISSUE THREAD

**Issue by Shemmmy** [AlexxIT/WebRTC#578]
Background music pauses on Android when loading a WebRTC card with muted sound
This happens when you launch the companion app HA.

Home Assistant 2023.10.1
WebRTC v3.3.0
HA app version: 2023.10.2-full

**Comment by AlexxIT** [comment]
Don't know if this can be fixed. Problem with only WebRTC. No problem with MSE. No problem if stream without audio

**Comment by rextroumbley** [comment]
I only experienced this same issue after updating from v3.3.0. My cameras microphones are disabled and I configured go2rtc with their video only RTSP streams.  Before updating to v3.4.0 or v3.5.0, my cards were configured to use "mode: webrtc" and would display video from my cameras without pausing the playback of any other Android apps. 

If I use App Ops to take audio focus permission away from the Companion App, the cards do not load. I have only been able to use the WebRTC cards without pausing other apps by changing the configuration to "mode: mse." However, MSE loads much slower and frequently lags on my phone and computer. Any other options to prevent audio pauses while using "mode: webrtc"?
